pkg/xray/resource_xray_security_policy_test.go

```diff
@@ -546,6 +649,36 @@ const securityPolicyCVSS = `resource "xray_security_policy" "{{ .resource_name }
 	}
 }`
 
+const securityPolicyCVSSMinSeverityMaliciousPkg = `resource "xray_security_policy" "{{ .resource_name }}" {
+	name = "{{ .policy_name }}"
+	description = "{{ .policy_description }}"
+	type = "security"
+	rule {
+		name 		= "{{ .rule_name }}"
+		priority 	= 1
+		criteria {
+			min_severity 	  = "{{ .min_severity }}"
+			malicious_package = "{{ .malicious_package }}"
```

Comment:
Boolean value doesn't require `"` wrapping around the value.

Reply:
Yup, it works both ways, but not required.